protokube/cmd/protokube/main.go

```diff
@@ -312,23 +282,23 @@ func run() error {
 	}
 
 	k := &protokube.KubeBoot{
-		Master:            master,
```

Comment:
I placed the master option back into the protokube (nodeup model and protokube binary) .. I wouldn't mind taking a stab and removing this functionality completely from protokube, but i'll leave it for another PR for now.